code/modules/clothing/neck/neck.dm

```diff
@@ -28,6 +28,31 @@
 	icon_state = "horribletie"
 	item_color = "horribletie"
 
+/obj/item/clothing/neck/translator
+	name = "translation collar"
+	desc = "A small translator fitted around one's neck. This one seems to be broken."
+	icon_state = "horribletie"
+	item_color = "horribletie"
+	var/language
+
+/obj/item/clothing/neck/translator/common
+	desc = "A small translator fitted around one's neck. This one is for Galactic Common."
+	language = /datum/language/common
+
+/obj/item/clothing/neck/translator/equipped(mob/user, slot)
+	if(!ishuman(user))
+		return
+	if((slot == slot_neck) && language)
+		var/mob/living/carbon/human/H = user
+		H.grant_language(language)
+
+/obj/item/clothing/neck/translator/dropped(mob/user)
+	if(!ishuman(user))
+		return
+	var/mob/living/carbon/human/H = user
+	if((H.get_item_by_slot(slot_neck) == src) && language)
+		H.remove_language(language)
```

Comment:
Wait. So if a non-ethari equipped this and then took it off, they would lose their access to galactic common? 🤔

There's code that can fix that inside the vox PR - if you were to restrict this to Ashwalkers and Ethari, it would be fine.

Reply:
Look at the TODO list nerd.